docs/src/main/asciidoc/logging.adoc

```diff
@@ -474,11 +473,10 @@ The Syslog server address and port are specified in the host and port properties
 +
 [source, properties]
 ----
-# Per-category Syslog handler
 quarkus.log.syslog.handlers = com.example.MyCategorySyslogHandler
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.level = WARN
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.host = syslog.example.com
-quarkus.log.syslog.com.example.MyCategorySyslogHandler.port = 514
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".level = WARN
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".host = syslog.example.com
+quarkus.log.syslog.com.example."MyCategorySyslogHandler".port = 514
```

Comment:
I don't think this is correct.

Reply:
@dmlloyd Hello David. Could I ask you for a review and maybe better Syslog examples?

Reply:
@Ladicek OK, thank you somuch, Laďa, for the examples! Patched the examples and pushed the commit.
Cheers

Reply:
Should be fixed now. I actually verified all the examples here using https://github.com/Ladicek/quarkus-tinies/tree/main/logging